src/main/io/osd.h

```diff
@@ -49,6 +49,9 @@ extern const char * const osdTimerSourceNames[OSD_NUM_TIMER_TYPES];
 
 // NB: to ensure backwards compatibility, new enum values must be appended at the end but before the OSD_XXXX_COUNT entry.
 
+// *** IMPORTANT ***
+// If you are adding additional elements that do not require any conditional display logic,
+// you must add the elements to the osdElementDisplayOrder[] array
```

Comment:
Please add the filename to look for `osdElementDisplayOrder[]` as well - if we can make it obvious, we shold.